test-docker-compose.yaml

```diff
@@ -11,7 +11,7 @@ services:
     build:
       context: 'docker/api'
       args:
-        - REQUIREMENTS=${REQUIREMENTS:-requirements.txt}
+        - REQUIREMENTS=${REQUIREMENTS:-requirements-dev.txt}
```

Comment:
What is the need of changing the requirements file here?

Reply:
It's explained in the commit message.